kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/poll/presenter/PollWebViewPresenter.java

```diff
@@ -70,7 +71,7 @@ private void setTitle(String title) {
 	}
 
 	private void createOrder() {
-		EarnOrderCreationRequested.create(EarnOrderCreationRequested.OfferType.fromValue(contentType), (double) amount, offerID);
+		eventLogger.send(EarnOrderCreationRequested.create(EarnOrderCreationRequested.OfferType.fromValue(contentType), (double) amount, offerID));
```

Comment:
EarnOrderCreationRequested.OfferType.fromValue(contentType) can throw runtime exception for example if we add new contentType in the future old SDK will crash because  OfferType.fromValue throws IllegalArgumentException if it receive a contentType not know - I think it safe to just pass the   contentType itself - worst case scenario we are in validation of the json schema but the event will be sent. This can be true to any other place we are using fromValue();